main.c

```diff
@@ -69,6 +69,22 @@ int main(void)
     MSS_TIM64_enable_irq();
     NVIC_SetPriority(Timer1_IRQn, 1);
 
+    /* load citi conf on startup */
+    uint32_t cfg_id_32 = 0;
+    mem_read(NVM_CITIROC_CONF_NO, &cfg_id_32);
+    uint8_t cfg_id = (uint8_t)(cfg_id_32);
+    uint8_t *nvm_cfg_addr =
+    (uint8_t *)(NVM_ADDR+CITIROC_OFFSET+(cfg_id*CITIROC_LEN));
+
+    if (cfg_id == nvm_cfg_addr[CITIROC_LEN-1]) {
+        mem_ram_write(RAM_CITI_CONF, nvm_cfg_addr);
+        citiroc_send_slow_control();
```

Comment:
I'm getting a compiler warning for an implicit declaration of this function. Please add an `#include` of the `citiroc.h` file.